src/service/ItemMapper.ts

```diff
@@ -11,7 +11,7 @@ export interface CheckDetailsItem {
     serviceName?: string;
     titleText?: string;
     pageTitle?: string;
-    happensNext?: string;
+    happensNext?: any;
```

Comment:
is there a reason this was changed to any type?

Reply:
I was writing an if block so that i will assign it to the happensNext but forgot to take it out when I took out the others.